components/.storybook/main.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-a11y"],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y",
+    "storybook-addon-designs",
```

Comment:
Intentional different reference path?

Reply:
Not sure I follow, these are package names. And `storybook-addon-designs` is not under the `@storybook` namespace.

Reply:
Sorry for poor phrasing - you answered what I was double checking 👍